packages/shared/lib/contexts/governance/utils/isAnyAccountVotingForSelectedProposal.ts

```diff
@@ -2,9 +2,8 @@ import { activeAccounts } from '@core/profile'
 import { get } from 'svelte/store'
 import { isVotingForSelectedProposal } from './isVotingForSelectedProposal'
 
-export async function isAnyAccountVotingForSelectedProposal(): Promise<boolean> {
+export function isAnyAccountVotingForSelectedProposal(): boolean {
     const accountIndexes = get(activeAccounts).map((account) => account.index)
-    const isVotingPromises = accountIndexes.map(isVotingForSelectedProposal)
-    const results = await Promise.all(isVotingPromises)
-    return results.some((bool) => bool === true)
+    const accountIsVoting = accountIndexes.map(isVotingForSelectedProposal)
+    return accountIsVoting.some((bool) => bool === true)
```

Comment:
I'm not sure that `accountIsVoting` is an accurate name for this data, but this could also just be one line with only iterating through the array once:
```typescript
return accountIndices.some(isVotingForSelectedProposal)
```